### ISSUE TITLE
Graphs don't show

### ISSUE THREAD

**Issue by BrunoSRibeiro** [kalkih/mini-graph-card#420]
Hello,

I'm using HA 0.113.3 and mini-graph 0.9.4.
The card shows the entity value but don't show the graph.

Follow the example the my card 
![card](https://user-images.githubusercontent.com/21297879/89802118-6ab93c80-db07-11ea-8cbf-73c3493b0ea6.jpg)

The module configuration 
![module](https://user-images.githubusercontent.com/21297879/89802033-4e1d0480-db07-11ea-8a16-04601bd462e2.jpg)

The path where mini-graph is hosted 
![path](https://user-images.githubusercontent.com/21297879/89802065-57a66c80-db07-11ea-90ba-60bb112a6288.jpg)

I use the addon Grafana, the graphs works.

Thx

**Comment by nzepp** [comment]
This looks the same as what I am experiencing in PR#419. If you hover over the top of the card are the graph points shown seemingly outside of the scale of the graph?

**Comment by c0rnflake** [comment]
Home Assistant 0.114.0 has the same issue.  No graphs being displayed with mini graph card 0.9.4

**Comment by kalkih** [comment]
Hey,
I'm on vacation a few more days so I'm not able to update HA to debug this right now.

I did however read the breaking changes and don't see anything that should affect the card.
There were however changes made to the recorder, so make sure HA is recording the entities you have in the cards correctly. If there's no history available for an entity you'll end up with an empty graph, like in your screenshots.
you could try setting up a normal history graph card with the same entity and see if data shows up there.

Let me know!

**Comment by BrunoSRibeiro** [comment]
Hello,

I removed the default_config: line in the configuration.yaml file after some warnings in last update.
I reinserted the default_config: line and the graphs works or if you don't want receive warnings about default_config: line, add history: line in the configuration.yaml.

Thx @kalkih

**Comment by boesing** [comment]
Hm, I am still experiencing this issue.
When I switch to dashboard edit-mode, the graph shows up.
When I close edit-mode, graphs stay until I switch to another dashboard.

I've made a video to show the problem. There are no errors in the developer console.
I am on latest home-assistant 0.118.4 and I got this for some months now. :-/ 

https://drive.google.com/file/d/15Xpkp70aNYugfP8MYw_y-zpelRFrdMsQ/view
